src/sentry/api_gateway/proxy.py

```diff
@@ -0,0 +1,54 @@
+"""
+Utilities related to proxying a request to a region silo
+"""
+
+from urllib.parse import urljoin
+
+import requests as external_request
+from requests import Response as external_response
+from rest_framework.exceptions import NotFound
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.utils import generate_region_url
+from sentry.models.organization import Organization
+
+
+def get_org_region(slug: str) -> str:
+    from sentry.types.region import get_region_for_organization
+
+    org = Organization.objects.get(slug=slug)
+    region_info = get_region_for_organization(org)
+    return region_info.address
```

Comment:
The method name implies it returns the entire `Region` object (so we can do that or change the name).